### ISSUE TITLE
Missed prime

### ISSUE THREAD

**Issue by opoudjis** [plurimath/plurimath#116]
`X'` in Asciimath is a convenience notation for X′, since of course ′ ″ are not Unicode characters.

This behaviour is not explicitly documented in asciimath.org, but it is in the examples:

`f'(x) = dy/dx` 

maps to:

````
<math xmlns="http://www.w3.org/1998/Math/MathML">
   <mstyle displaystyle="true">
     <mi>f</mi>
     <mo>&#x2032;</mo>
     <mrow>
       <mo>(</mo>
       <mi>x</mi> 
      <mo>)</mo>
     </mrow>
     <mo>=</mo>
     <mfrac>
       <mrow>
         <mi>d</mi>
         <mi>y</mi>
       </mrow>
       <mrow>
         <mi>d</mi>
         <mi>x</mi>
       </mrow>
     </mfrac>
   </mstyle>
 </math>
````

This mapping is in asciidoc/asciimath

https://github.com/asciidoctor/asciimath/blob/e52761299166019df0c6a1f5e714395c1e859530/lib/asciimath/parser.rb#L327

Puzzlingly, double prime is nowhere to be found. I think we will need to do it eventually anyway, and that `''` (two apostrophes) are safer for double prime than `"` (straight quote)

This is in a metanorma-standoc test case.

**Comment by opoudjis** [comment]
Expected behaviour:

Asciimath `bar X'`

Mathml:

````
<mover accent="true">
                           <mrow>
                             <mi>X</mi>
                           </mrow>
                             <mo>¯</mo>
                         </mover>
                         <mo>′</mo>

````

**Comment by ronaldtse** [comment]
@suleman-uzair can you please help fix this ASAP and issue a new version? Thanks.

**Comment by suleman-uzair** [comment]
@opoudjis released new version(**0.2.3**) of **Plurimath**.
